.github/build_n_deploy_prod.yaml

```diff
@@ -0,0 +1,18 @@
+name: Buld and deploy prod
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+jobs:
+  sanity-deploy:
+    runs-on: ubuntu-18.04
```

Comment:
Burde vi kjøre på ubuntu-latest?

Reply:
Jeg så bare på docen til actionen https://github.com/marketplace/actions/github-action-for-sanity-io, funker sitter fint med latest